homeassistant/components/broadlink/remote.py

```diff
@@ -73,6 +74,7 @@
     {
         vol.Required(CONF_HOST): vol.All(vol.Any(hostname, ip_address), cv.string),
         vol.Required(CONF_MAC): mac_address,
+        vol.Optional(CONF_TYPE, default=DEFAULT_TYPE): int,
```

Comment:
Which types are permitted and can we use anything from config_validation e.g. `cv.positive_int`?

Reply:
The supported types are [here](https://github.com/mjg59/python-broadlink/blob/master/broadlink/__init__.py). But I don't think validating them is a good idea, as Broadlink often launches new types of devices and this code would quickly become obsolete. I think that validating an integer is the most appropriate solution, since the device type is presented in hex.

Reply:
Absolutely agree that it would be difficult to validate each available device type. However, we should make sure that this validator matches the device types e.g. all device types are positive as far as I can see. Therefore, the validator should be starting from zero and upwards.

Reply:
Done.